attestation-verifier/utils/tools/containers/nats/Dockerfile

```diff
@@ -3,26 +3,47 @@
 # * SPDX-License-Identifier: BSD-3-Clause
 # */
 
-FROM debian:bookworm-slim@sha256:6ac2c08566499cc2415926653cf2ed7c3aedac445675a013cc09469c9e118fdd
+# Multi-stage Dockerfile: build cfssl/cfssljson in a builder stage, copy only binaries into final image
+FROM golang:1.24.7-bookworm AS builder
 
-COPY nats/download-tls-certs.sh /download-tls-certs.sh
-COPY nats/entrypoint.sh /entrypoint.sh
+# allow optional CFSSL_TAG override at build time; if empty, resolve latest
+ARG CFSSL_TAG
+RUN set -eux; \
+    apt-get update && apt-get install -y --no-install-recommends git curl ca-certificates && \
+    CFSSL_TAG="${CFSSL_TAG:-$(curl -fsSL -o /dev/null -w '%{url_effective}' https://github.com/cloudflare/cfssl/releases/latest | sed -E 's:.*/::')}" && \
```

Comment:
we have to build cfssl tool from source due to the prebuilt version is out of date with many CVE against it due to goland dependencies, hence until cfssl project updates the prebuild binary we have to build from source with latest golang dependencies pulled.